### PR TITLE
Manual updates 20220215 split FB

### PR DESCRIPTION
**Pull request by moljac**
### Does this change any of the generated binding API's?

Yes. Updates, new dependencies, etc

### Describe your contribution

PR split

https://github.com/xamarin/GooglePlayServicesComponents/pull/575

- com.google.firebase:firebase-abt - 21.0.0 -> 21.0.1
- com.google.firebase:firebase-ads - 20.3.0 -> 20.4.0
- com.google.firebase:firebase-adslite - 20.3.0 -> 20.4.0
- com.google.firebase:firebase-annotations - 16.0.0 -> 16.1.0
- com.google.firebase:firebase-appcheck-interop - 16.0.0beta01 -> 16.0.0beta04
- com.google.firebase:firebase-auth - 21.0.1 -> 21.0.3
- com.google.firebase:firebase-config - 21.0.1 -> 21.0.2
- com.google.firebase:firebase-core - 19.0.1 -> 19.0.2
- com.google.firebase:firebase-crashlytics - 18.2.1 -> 18.2.10
- com.google.firebase:firebase-crashlytics-ndk - 18.2.1 -> 18.2.10
- com.google.firebase:firebase-database - 20.0.0 -> 20.0.4
- com.google.firebase:firebase-database-collection - 18.0.0 -> 18.0.1
- com.google.firebase:firebase-datatransport - 18.0.1 -> 18.1.2
- com.google.firebase:firebase-dynamiclinks - 20.1.1 -> 21.0.1
- com.google.firebase:firebase-firestore - 23.0.3 -> 24.0.2
- com.google.firebase:firebase-functions - 20.0.0 -> 20.0.2
- com.google.firebase:firebase-inappmessaging - 20.1.0 -> 20.1.2
- com.google.firebase:firebase-inappmessaging-display - 20.1.0 -> 20.1.2
- com.google.firebase:firebase-measurement-connector - 19.0.0 -> 20.0.0
- com.google.firebase:firebase-messaging - 22.0.0 -> 23.0.3
- com.google.firebase:firebase-perf - 20.0.2 -> 20.0.5
- com.google.android.ump:user-messaging-platform -  -> 1.0.0
- com.google.android.datatransport:transport-backend-cct - 3.0.0 -> 3.1.2
- com.google.android.datatransport:transport-runtime - 3.0.1 -> 3.1.2
- com.google.auto.value:autovalue-annotations -  -> 1.9.0
- com.google.code.gson:gson -  -> 2.9.0
- com.google.firebase:firebase-analytics -  -> 16.3.0
- com.google.guava:guava - 31.1-android -> 31.0.1android
- com.google.protobuf:protobuf-javalite -  -> 3.19.2

### CODE REVIEW

**Comment by moljac** [comment]
Blocked by dependencies and GPS updates needed to update FB packages:

```
Dependency errors : 16
1
	System.Exception: 
No matching artifact config found for: 
			com.google.android.gms.play-services-ads:20.5.0
to satisfy dependency of: 
			com.google.firebase.firebase-ads:20.5.0

	Please add following json snippet to config.json:


      {
        "groupId": "com.google.android.gms",
        "artifactId": "play-services-ads",
        "version": "20.5.0",
        "nugetVersion": "CHECK PREFIX 20.5.0",
        "nugetId": "CHECK NUGET ID",
        "dependencyOnly": true/false
      }
						


2
	System.Exception: 
No matching artifact config found for: 
			com.google.android.gms.play-services-ads-lite:20.5.0
to satisfy dependency of: 
			com.google.firebase.firebase-ads-lite:20.5.0

	Please add following json snippet to config.json:


      {
        "groupId": "com.google.android.gms",
        "artifactId": "play-services-ads-lite",
        "version": "20.5.0",
        "nugetVersion": "CHECK PREFIX 20.5.0",
        "nugetId": "CHECK NUGET ID",
        "dependencyOnly": true/false
      }
						


3
	System.Exception: 
No matching artifact config found for: 
			com.google.android.gms.play-services-measurement:19.0.2
to satisfy dependency of: 
			com.google.firebase.firebase-analytics:19.0.2

	Please add following json snippet to config.json:


      {
        "groupId": "com.google.android.gms",
        "artifactId": "play-services-measurement",
        "version": "19.0.2",
        "nugetVersion": "CHECK PREFIX 19.0.2",
        "nugetId": "CHECK NUGET ID",
        "dependencyOnly": true/false
      }
						


4
	System.Exception: 
No matching artifact config found for: 
			com.google.android.gms.play-services-measurement-api:19.0.2
to satisfy dependency of: 
			com.google.firebase.firebase-analytics:19.0.2

	Please add following json snippet to config.json:


      {
        "groupId": "com.google.android.gms",
        "artifactId": "play-services-measurement-api",
        "version": "19.0.2",
        "nugetVersion": "CHECK PREFIX 19.0.2",
        "nugetId": "CHECK NUGET ID",
        "dependencyOnly": true/false
      }
						


5
	System.Exception: 
No matching artifact config found for: 
			com.google.android.gms.play-services-measurement-sdk:19.0.2
to satisfy dependency of: 
			com.google.firebase.firebase-analytics:19.0.2

	Please add following json snippet to config.json:


      {
        "groupId": "com.google.android.gms",
        "artifactId": "play-services-measurement-sdk",
        "version": "19.0.2",
        "nugetVersion": "CHECK PREFIX 19.0.2",
        "nugetId": "CHECK NUGET ID",
        "dependencyOnly": true/false
      }
						


6
	System.Exception: 
No matching artifact config found for: 
			com.google.android.datatransport.transport-backend-cct:3.1.2
to satisfy dependency of: 
			com.google.firebase.firebase-crashlytics:18.2.8

	Please add following json snippet to config.json:


      {
        "groupId": "com.google.android.datatransport",
        "artifactId": "transport-backend-cct",
        "version": "3.1.2",
        "nugetVersion": "CHECK PREFIX 3.1.2",
        "nugetId": "CHECK NUGET ID",
        "dependencyOnly": true/false
      }
						


7
	System.Exception: 
No matching artifact config found for: 
			com.google.android.datatransport.transport-runtime:3.1.2
to satisfy dependency of: 
			com.google.firebase.firebase-crashlytics:18.2.8

	Please add following json snippet to config.json:


      {
        "groupId": "com.google.android.datatransport",
        "artifactId": "transport-runtime",
        "version": "3.1.2",
        "nugetVersion": "CHECK PREFIX 3.1.2",
        "nugetId": "CHECK NUGET ID",
        "dependencyOnly": true/false
      }
						


8
	System.Exception: 
No matching artifact config found for: 
			com.google.android.datatransport.transport-backend-cct:3.1.1
to satisfy dependency of: 
			com.google.firebase.firebase-datatransport:18.1.1

	Please add following json snippet to config.json:


      {
        "groupId": "com.google.android.datatransport",
        "artifactId": "transport-backend-cct",
        "version": "3.1.1",
        "nugetVersion": "CHECK PREFIX 3.1.1",
        "nugetId": "CHECK NUGET ID",
        "dependencyOnly": true/false
      }
						


9
	System.Exception: 
No matching artifact config found for: 
			com.google.android.datatransport.transport-runtime:3.1.1
to satisfy dependency of: 
			com.google.firebase.firebase-datatransport:18.1.1

	Please add following json snippet to config.json:


      {
        "groupId": "com.google.android.datatransport",
        "artifactId": "transport-runtime",
        "version": "3.1.1",
        "nugetVersion": "CHECK PREFIX 3.1.1",
        "nugetId": "CHECK NUGET ID",
        "dependencyOnly": true/false
      }
						


10
	System.Exception: 
No matching artifact config found for: 
			io.grpc.grpc-android:1.41.0
to satisfy dependency of: 
			com.google.firebase.firebase-firestore:24.0.1

	Please add following json snippet to config.json:


      {
        "groupId": "io.grpc",
        "artifactId": "grpc-android",
        "version": "1.41.0",
        "nugetVersion": "CHECK PREFIX 1.41.0",
        "nugetId": "CHECK NUGET ID",
        "dependencyOnly": true/false
      }
						


11
	System.Exception: 
No matching artifact config found for: 
			io.grpc.grpc-okhttp:1.41.0
to satisfy dependency of: 
			com.google.firebase.firebase-firestore:24.0.1

	Please add following json snippet to config.json:


      {
        "groupId": "io.grpc",
        "artifactId": "grpc-okhttp",
        "version": "1.41.0",
        "nugetVersion": "CHECK PREFIX 1.41.0",
        "nugetId": "CHECK NUGET ID",
        "dependencyOnly": true/false
      }
						


12
	System.Exception: 
No matching artifact config found for: 
			io.grpc.grpc-protobuf-lite:1.41.0
to satisfy dependency of: 
			com.google.firebase.firebase-firestore:24.0.1

	Please add following json snippet to config.json:


      {
        "groupId": "io.grpc",
        "artifactId": "grpc-protobuf-lite",
        "version": "1.41.0",
        "nugetVersion": "CHECK PREFIX 1.41.0",
        "nugetId": "CHECK NUGET ID",
        "dependencyOnly": true/false
      }
						


13
	System.Exception: 
No matching artifact config found for: 
			io.grpc.grpc-stub:1.41.0
to satisfy dependency of: 
			com.google.firebase.firebase-firestore:24.0.1

	Please add following json snippet to config.json:


      {
        "groupId": "io.grpc",
        "artifactId": "grpc-stub",
        "version": "1.41.0",
        "nugetVersion": "CHECK PREFIX 1.41.0",
        "nugetId": "CHECK NUGET ID",
        "dependencyOnly": true/false
      }
						


14
	System.Exception: 
No matching artifact config found for: 
			io.grpc.grpc-okhttp:1.41.0
to satisfy dependency of: 
			com.google.firebase.firebase-inappmessaging:20.1.1

	Please add following json snippet to config.json:


      {
        "groupId": "io.grpc",
        "artifactId": "grpc-okhttp",
        "version": "1.41.0",
        "nugetVersion": "CHECK PREFIX 1.41.0",
        "nugetId": "CHECK NUGET ID",
        "dependencyOnly": true/false
      }
						


15
	System.Exception: 
No matching artifact config found for: 
			io.grpc.grpc-protobuf-lite:1.41.0
to satisfy dependency of: 
			com.google.firebase.firebase-inappmessaging:20.1.1

	Please add following json snippet to config.json:


      {
        "groupId": "io.grpc",
        "artifactId": "grpc-protobuf-lite",
        "version": "1.41.0",
        "nugetVersion": "CHECK PREFIX 1.41.0",
        "nugetId": "CHECK NUGET ID",
        "dependencyOnly": true/false
      }
						


16
	System.Exception: 
No matching artifact config found for: 
			io.grpc.grpc-stub:1.41.0
to satisfy dependency of: 
			com.google.firebase.firebase-inappmessaging:20.1.1

	Please add following json snippet to config.json:


      {
        "groupId": "io.grpc",
        "artifactId": "grpc-stub",
        "version": "1.41.0",
        "nugetVersion": "CHECK PREFIX 1.41.0",
        "nugetId": "CHECK NUGET ID",
        "dependencyOnly": true/false
      }				
``` 

Note: DataTransport 3.1.2 depends on `firebase-encoders-proto`, so dependency cycle.

**Comment by moljac** [comment]
Missing dependencies

nuget restore error:

```
./generated/com.google.firebase.firebase-inappmessaging/com.google.firebase.firebase-inappmessaging.csproj : error NU1102: Unable to find package Xamarin.Protobuf.JavaLite with version (>= 3.17.2) [./generated/GooglePlayServices.sln]
./generated/com.google.firebase.firebase-inappmessaging/com.google.firebase.firebase-inappmessaging.csproj : error NU1102:   - Found 4 version(s) in nuget.org [ Nearest version: 3.14.0.3 ] [./generated/GooglePlayServices.sln]
./generated/com.google.firebase.firebase-inappmessaging/com.google.firebase.firebase-inappmessaging.csproj : error NU1102:   - Found 0 version(s) in /usr/local/share/dotnet/library-packs [./generated/GooglePlayServices.sln]
```

**Comment by moljac** [comment]
samples error:

```
  MSBUILD : java error JAVA0000: Error in obj/Debug/lp/150/jl/classes.jar:com/google/android/gms/measurement/internal/zzga.class: [/Users/Shared/Projects/d/X/GPS-FB-MLKit/mu-20220215-split-fb/samples/com.google.firebase/firebase-ads/FirebaseAdmobQuickstart/FirebaseAdmobQuickstart.csproj]
MSBUILD : java error JAVA0000: Type com.google.android.gms.measurement.internal.zzga is defined multiple times: obj/Debug/lp/150/jl/classes.jar:com/google/android/gms/measurement/internal/zzga.class, obj/Debug/lp/161/jl/classes.jar:com/google/android/gms/measurement/internal/zzga.class [/Users/Shared/Projects/d/X/GPS-FB-MLKit/mu-20220215-split-fb/samples/com.google.firebase/firebase-ads/FirebaseAdmobQuickstart/FirebaseAdmobQuickstart.csproj]
MSBUILD : java error JAVA0000: Compilation failed [/Users/Shared/Projects/d/X/GPS-FB-MLKit/mu-20220215-split-fb/samples/com.google.firebase/firebase-ads/FirebaseAdmobQuickstart/FirebaseAdmobQuickstart.csproj]
MSBUILD : java error JAVA0000: java.lang.RuntimeException: com.android.tools.r8.CompilationFailedException: Compilation failed to complete, origin: obj/Debug/lp/150/jl/classes.jar [/Users/Shared/Projects/d/X/GPS-FB-MLKit/mu-20220215-split-fb/samples/com.google.firebase/firebase-ads/FirebaseAdmobQuickstart/FirebaseAdmobQuickstart.csproj]
```

**Comment by moljac** [comment]
```
- com.google.firebase:firebase-abt - 21.0.0 -> 21.0.1
- com.google.firebase:firebase-ads - 20.3.0 -> 20.4.0
- com.google.firebase:firebase-adslite - 20.3.0 -> 20.4.0
- com.google.firebase:firebase-annotations - 16.0.0 -> 16.1.0
- com.google.firebase:firebase-appcheckinterop - 16.0.0beta01 -> 16.0.0beta04
- com.google.firebase:firebase-auth - 21.0.1 -> 21.0.3
- com.google.firebase:firebase-config - 21.0.1 -> 21.0.2
- com.google.firebase:firebase-core - 19.0.1 -> 19.0.2
- com.google.firebase:firebase-crashlytics - 18.2.1 -> 18.2.10
- com.google.firebase:firebase-crashlytics-ndk - 18.2.1 -> 18.2.10
- com.google.firebase:firebase-database - 20.0.0 -> 20.0.4
- com.google.firebase:firebase-database-collection - 18.0.0 -> 18.0.1
- com.google.firebase:firebase-datatransport - 18.0.1 -> 18.1.2
- com.google.firebase:firebase-dynamiclinks - 20.1.1 -> 21.0.1
- com.google.firebase:firebase-firestore - 23.0.3 -> 24.0.2
- com.google.firebase:firebase-functions - 20.0.0 -> 20.0.2
- com.google.firebase:firebase-inappmessaging - 20.1.0 -> 20.1.2
- com.google.firebase:firebase-inappmessaging-display - 20.1.0 -> 20.1.2
- com.google.firebase:firebase-measurementconnector - 19.0.0 -> 20.0.0
- com.google.firebase:firebase-messaging - 22.0.0 -> 23.0.3
- com.google.firebase:firebase-perf - 20.0.2 -> 20.0.5
- com.google.android.ump:user-messaging-platform -  -> 1.0.0
- com.google.android.datatransport:transport-backend-cct - 3.0.0 -> 3.1.2
- com.google.android.datatransport:transport-runtime - 3.0.1 -> 3.1.2
- com.google.auto.value:autovalue-annotations -  -> 1.9.0
- com.google.code.gson:gson -  -> 2.9.0
- com.google.firebase:firebase-analytics -  -> 16.3.0
- com.google.guava:guava - 31.1-android -> 31.0.1android
- com.google.protobuf:protobuf-javalite -  -> 3.19.2
```

File: config.json [hunk 1]
```diff
@@ -1908,11 +1909,35 @@
...
+        "artifactId": "firebase-analytics",
+        "version": "16.3.0",
+        "nugetVersion": "116.3.0",
+        "nugetId": "Xamarin.Firebase.Analytics",
+        "dependencyOnly": true
+      },
       {
         "groupId": "com.google.guava",
         "artifactId": "guava",
-        "version": "31.1-android",
```
Comment (jpobst) [inline config.json:1914]: Package downgrade.

If this is required, please provide information about why it is required.  If not, we should use the most recent version.

**Review by jpobst — COMMENTED** [review]
Just one small issue.

File: config.json (hunk 1 above)
Comment (jonathanpeppers) [reply config.json:1914]: @moljac did you address this one?

Comment (moljac) [reply config.json:1914]: yes

with latest:

```
/Users/Shared/Projects/d/X/GPS-FB-MLKit/mu-20220215-split-fb/generated/com.google.firebase.firebase-firestore/com.google.firebase.firebase-firestore.csproj : error NU1605: Detected package downgrade: Xamarin.Google.Guava from 31.1.0.3 to 31.1.0. Reference the package directly from the project to select a different version.  [/Users/Shared/Projects/d/X/GPS-FB-MLKit/mu-20220215-split-fb/generated/GooglePlayServices.sln]
/Users/Shared/Projects/d/X/GPS-FB-MLKit/mu-20220215-split-fb/generated/com.google.firebase.firebase-firestore/com.google.firebase.firebase-firestore.csproj : error NU1605:  Xamarin.Firebase.Firestore -> Xamarin.Grpc.Android -> Xamarin.Google.Guava (>= 31.1.0.3)  [/Users/Shared/Projects/d/X/GPS-FB-MLKit/mu-20220215-split-fb/generated/GooglePlayServices.sln]
/Users/Shared/Projects/d/X/GPS-FB-MLKit/mu-20220215-split-fb/generated/com.google.firebase.firebase-firestore/com.google.firebase.firebase-firestore.csproj : error NU1605:  Xamarin.Firebase.Firestore -> Xamarin.Google.Guava (>= 31.1.0) [/Users/Shared/Projects/d/X/GPS-FB-MLKit/mu-20220215-split-fb/generated/GooglePlayServices.sln]
/Users/Shared/Projects/d/X/GPS-FB-MLKit/mu-20220215-split-fb/generated/com.google.firebase.firebase-firestore/com.google.firebase.firebase-firestore.csproj : error NU1605: Detected package downgrade: Xamarin.Google.Guava from 31.1.0.3 to 31.1.0. Reference the package directly from the project to select a different version.  [/Users/Shared/Projects/d/X/GPS-FB-MLKit/mu-20220215-split-fb/generated/GooglePlayServices.sln]
/Users/Shared/Projects/d/X/GPS-FB-MLKit/mu-20220215-split-fb/generated/com.google.firebase.firebase-firestore/com.google.firebase.firebase-firestore.csproj : error NU1605:  Xamarin.Firebase.Firestore -> Xamarin.Grpc.OkHttp -> Xamarin.Google.Guava (>= 31.1.0.3)  [/Users/Shared/Projects/d/X/GPS-FB-MLKit/mu-20220215-split-fb/generated/GooglePlayServices.sln]
/Users/Shared/Projects/d/X/GPS-FB-MLKit/mu-20220215-split-fb/generated/com.google.firebase.firebase-firestore/com.google.firebase.firebase-firestore.csproj : error NU1605:  Xamarin.Firebase.Firestore -> Xamarin.Google.Guava (>= 31.1.0) [/Users/Shared/Projects/d/X/GPS-FB-MLKit/mu-20220215-split-fb/generated/GooglePlayServices.sln]
/Users/Shared/Projects/d/X/GPS-FB-MLKit/mu-20220215-split-fb/generated/com.google.firebase.firebase-firestore/com.google.firebase.firebase-firestore.csproj : error NU1605: Detected package downgrade: Xamarin.Google.Guava from 31.1.0.3 to 31.1.0. Reference the package directly from the project to select a different version.  [/Users/Shared/Projects/d/X/GPS-FB-MLKit/mu-20220215-split-fb/generated/GooglePlayServices.sln]
/Users/Shared/Projects/d/X/GPS-FB-MLKit/mu-20220215-split-fb/generated/com.google.firebase.firebase-firestore/com.google.firebase.firebase-firestore.csproj : error NU1605:  Xamarin.Firebase.Firestore -> Xamarin.Grpc.Protobuf.Lite -> Xamarin.Google.Guava (>= 31.1.0.3)  [/Users/Shared/Projects/d/X/GPS-FB-MLKit/mu-20220215-split-fb/generated/GooglePlayServices.sln]
/Users/Shared/Projects/d/X/GPS-FB-MLKit/mu-20220215-split-fb/generated/com.google.firebase.firebase-firestore/com.google.firebase.firebase-firestore.csproj : error NU1605:  Xamarin.Firebase.Firestore -> Xamarin.Google.Guava (>= 31.1.0) [/Users/Shared/Projects/d/X/GPS-FB-MLKit/mu-20220215-split-fb/generated/GooglePlayServices.sln]
/Users/Shared/Projects/d/X/GPS-FB-MLKit/mu-20220215-split-fb/generated/com.google.firebase.firebase-firestore/com.google.firebase.firebase-firestore.csproj : error NU1605: Detected package downgrade: Xamarin.Google.Guava from 31.1.0.3 to 31.1.0. Reference the package directly from the project to select a different version.  [/Users/Shared/Projects/d/X/GPS-FB-MLKit/mu-20220215-split-fb/generated/GooglePlayServices.sln]
/Users/Shared/Projects/d/X/GPS-FB-MLKit/mu-20220215-split-fb/generated/com.google.firebase.firebase-firestore/com.google.firebase.firebase-firestore.csproj : error NU1605:  Xamarin.Firebase.Firestore -> Xamarin.Grpc.Stub -> Xamarin.Google.Guava (>= 31.1.0.3)  [/Users/Shared/Projects/d/X/GPS-FB-MLKit/mu-20220215-split-fb/generated/GooglePlayServices.sln]
/Users/Shared/Projects/d/X/GPS-FB-MLKit/mu-20220215-split-fb/generated/com.google.firebase.firebase-firestore/com.google.firebase.firebase-firestore.csproj : error NU1605:  Xamarin.Firebase.Firestore -> Xamarin.Google.Guava (>= 31.1.0) [/Users/Shared/Projects/d/X/GPS-FB-MLKit/mu-20220215-split-fb/generated/GooglePlayServices.sln]

```

File: config.json [hunk 2]
```diff
@@ -660,16 +660,16 @@
...
         "nugetId": "Xamarin.Firebase.Annotations",
         "dependencyOnly": false
       },
       {
         "groupId": "com.google.firebase",
         "artifactId": "firebase-appcheck-interop",
-        "version": "16.0.0-beta01",
-        "nugetVersion": "116.0.0.7-beta01",
+        "version": "16.0.0-beta04",
+        "nugetVersion": "116.0.0.4-beta04",
```
Comment (jpobst) [inline config.json:672]: I think this is a version downgrade:
`116.0.0.4-beta04` is considered "before" `116.0.0.7-beta01`.

This should probably be `116.0.0.7-beta04`.

File: config.json [hunk 3]
```diff
@@ -1305,6 +1305,15 @@
         "nugetId": "Xamarin.Android.Volley.CroNet",
         "dependencyOnly": false
       },
+      {
```
Comment (jpobst) [inline config.json:1308]: This entry is duplicated, version `2.0.0.2` is also in this file.

Comment (moljac) [reply config.json:1308]: seems like manual merge error

Comment (moljac) [reply config.json:1308]: fixed

File: config.json (hunk 2 above)
Comment (moljac) [reply config.json:672]: fixed

**Review by jpobst — COMMENTED** [review]
I don't think you committed your changes?

File: config.json (hunk 1 above)
Comment (moljac) [reply config.json:1912]: https://github.com/xamarin/GooglePlayServicesComponents/blob/mu-20220215-split-fb/config.json#L1934-L1941

```
      {
        "groupId": "com.google.guava",
        "artifactId": "guava",
        "version": "31.1.0-android",
        "nugetVersion": "31.1.0.3",
        "nugetId": "Xamarin.Google.Guava",
        "dependencyOnly": true
      },
```

Latest nuget:

https://www.nuget.org/packages/Xamarin.Google.Guava/31.1.0.3

**Comment by moljac** [comment]
> I don't think you committed your changes?

Checked. Everything was committed and pushed. Guava from "Just small issue" is `31.1.0-android`

**Comment by jpobst** [comment]
If I look at `config.json` on this branch:

- `Xamarin.Firebase.AppCheck.Interop` is still version `116.0.0.4-beta04`:
  - https://github.com/xamarin/GooglePlayServicesComponents/blob/mu-20220215-split-fb/config.json#L672
- The `1.0.0.4` entry for `Xamarin.Google.UserMessagingPlatform` is still present:
  - https://github.com/xamarin/GooglePlayServicesComponents/blob/mu-20220215-split-fb/config.json#L1309

**Comment by moljac** [comment]
> If I look at `config.json` on this branch:
> 
> * `Xamarin.Firebase.AppCheck.Interop` is still version `116.0.0.4-beta04`:
>   
>   * https://github.com/xamarin/GooglePlayServicesComponents/blob/mu-20220215-split-fb/config.json#L672
> * The `1.0.0.4` entry for `Xamarin.Google.UserMessagingPlatform` is still present:
>   
>   * https://github.com/xamarin/GooglePlayServicesComponents/blob/mu-20220215-split-fb/config.json#L1309

I had .NET7 preview issues last week and maybe I revereted before committing and pushing. Let me check.

**Comment by moljac** [comment]
fixed

**Review by jpobst — APPROVED** [review]

**Pull request merged**